workspaces/core/src/tagsInput/private/types/TagsInputRootProps.ts

```diff
@@ -0,0 +1,20 @@
+import Size from "./Size";
+
+export interface TagsInputRootProps
+  extends Omit<
+    React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement>,
+    'children' | 'ref'
+  > {
+  children?: React.ReactNode;
+  label?: string;
```

Comment:
Perhaps it makes sense to set "label" as  `JSX.Element | string;` for example for translation component